doc/sedcli.8

```diff
@@ -81,6 +81,15 @@ Changes the lock state for the device. Following access modes are available:
 .IP "\fB\-P -d <device>\fR or \fB\-\-set-password --device <device>\fR"
 Updates password for Admin1 authority in Locking SP.
 
+.IP "\fB\-M -d <device> -e <enable> -m <done>\fR or \fB\-\-mbr-control --device <device> --enable {TRUE|FALSE} --done {TRUE|FALSE}\fR"
+.IP
+Enable/Disable MBR Shadow and Set/Unset MBR Done.
```

Comment:
I think it should be:
"Enable/Disable MBR Shadow and/or Set/Unset MBR Done." because user can do both or either one, correct?

"MBR Done is TRUE only when the MBR Shadow is Enabled." -> "MBR Done update to TRUE will only take effect when MBR Shadow is enabled."